pkg/controllers/node/termination/metrics.go

```diff
@@ -38,4 +46,39 @@ var (
 		},
 		[]string{metrics.NodePoolLabel},
 	)
+	NodeLifetimeDurationSeconds = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: metrics.Namespace,
+			Subsystem: metrics.NodeSubsystem,
+			Name:      "lifetime_duration_seconds",
+			Help:      "The lifetime duration of the nodes since creation.",
+			Buckets: []float64{
```

Comment:
I created these bucket thresholds to show if there churn at the beginning of the node's life (within an hour), an idea of long lived nodes (that live for days), and some distribution of where everything fits in between.